homeassistant/components/refoss/config_flow.py

```diff
@@ -0,0 +1,36 @@
+"""ConfigFlow for Refoss."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from refoss_ha.util import get_mac_address
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_MAC
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+
+class RefossConfigFlow(ConfigFlow, domain=DOMAIN):
+    """RefossConfigFlow for Refoss."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """async_step_user for Refoss."""
+        mac = get_mac_address()
```

Comment:
This needs to be the mac address of the device.  

https://developers.home-assistant.io/docs/config_entries_config_flow_handler?_highlight=unique&_highlight=id&_highlight=re#unique-id-requirements

Reply:
in config_flow.py， Device information has not been obtained yet.     
Can I use the longitude and latitude of the host？

Reply:
Use discovery or ask for the ip address of the device, than connect and get the mac address.

Examples in the yeelight and tplink config flows

Reply:
i adopting Tomorowio's,  Using latitude and longitude  as unique_id